[mqtt] Errors in the log file after MQTT binding installation

Hi guys! (@davidgraeff)

I'm trying the latest openHAB 2.5.0 SNAPSHOT version and I have found the following errors after installing the MQTT binding:

```
2019-03-26 15:34:17.609 [ERROR] [org.openhab.binding.mqtt.generic    ] - bundle org.openhab.binding.mqtt.generic:2.5.0.201903260114 (223)[org.openhab.binding.mqtt.MqttChannelStateDescriptionProvider(215)] : Could not load implementation object class org.openhab.binding.mqtt.MqttChannelStateDescriptionProvider
java.lang.ClassNotFoundException: org.openhab.binding.mqtt.MqttChannelStateDescriptionProvider cannot be found by org.openhab.binding.mqtt.generic_2.5.0.201903260114
```

Many thanks for your work and support!

PS: The long version of the error message can be found here: https://pastebin.com/14RHjbd5

--- a/features/karaf/openhab-addons/src/main/feature/feature.xml
+++ b/features/karaf/openhab-addons/src/main/feature/feature.xml
@@ -495,9 +495,9 @@
         <feature>openhab-runtime-base</feature>
         <feature>openhab-transport-mqtt</feature>
         <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.mqtt/${project.version}</bundle>
-        <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.mqtt.generic/${project.version}</bundle>
-        <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.mqtt.homeassistant/${project.version}</bundle>
-        <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.mqtt.homie/${project.version}</bundle>
+        <bundle start-level="81">mvn:org.openhab.addons.bundles/org.openhab.binding.mqtt.generic/${project.version}</bundle>
+        <bundle start-level="82">mvn:org.openhab.addons.bundles/org.openhab.binding.mqtt.homeassistant/${project.version}</bundle>
+        <bundle start-level="82">mvn:org.openhab.addons.bundles/org.openhab.binding.mqtt.homie/${project.version}</bundle>
     </feature>
 
     <feature name="openhab-binding-nanoleaf" description="Nanoleaf Binding" version="${project.version}">

